src/backend/gl/src/window/web.rs

```diff
@@ -13,7 +13,7 @@ pub struct Swapchain {
     pub(crate) extent: window::Extent2D,
     pub(crate) channel: f::ChannelType,
     pub(crate) raw_format: native::TextureFormat,
-    pub(crate) frame_buffers: ArrayVec<[native::RawFrameBuffer; 3]>,
+    pub(crate) frame_buffers: ArrayVec<[native::RawFramebuffer; 3]>,
```

Comment:
-  `frame_buffers` -> `framebuffers`. Just to keep consistency with the rest.
- Do we need both ArrayVec and SmallVec in the codebase? Could we remove one of them?
- Why `3`? We are only using the first one.

Reply:
> Do we need both ArrayVec and SmallVec in the codebase? Could we remove one of them?

That's something I direly wanted to do. In particular, removing `SmallVec`. But it turns out that it serves a distinct use case that we still need in a few places.